src/types/index.ts

```diff
@@ -251,58 +216,52 @@ export enum ClaimType {
   InvestorUniquenessV2 = 'InvestorUniquenessV2',
 }
 
+export type AccreditedClaim = { type: ClaimType.Accredited; scope: Scope };
+
+export type AffiliateClaim = { type: ClaimType.Affiliate; scope: Scope };
+
+export type BuyLockupClaim = { type: ClaimType.BuyLockup; scope: Scope };
+
+export type SellLockupClaim = { type: ClaimType.SellLockup; scope: Scope };
+
 export type CddClaim = { type: ClaimType.CustomerDueDiligence; id: string };
 
+export type KnowYourCustomerClaim = { type: ClaimType.KnowYourCustomer; scope: Scope };
```

Comment:
Maybe `KycClaim`? Otherwise the other one should be `CustomerDueDiligenceClaim`

Reply:
I went with the long form since it looked more consistent.